isazonov/src/main/java/com/evilcorp/hcf/HibernateCustomFunctionsApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.evilcorp.hcf;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class HibernateCustomFunctionsApplication {
```

Comment:
I would the class since hql custom functions is not the only example you would have in the maven module

Reply:
No, that does not make much sense. We will confuse ourselves and other with this. Just name each package appropriately.

Reply:
Application renamed